keyboards/tweetydabird/lbs4/info.json

```
@@ -0,0 +1,76 @@
+{
+	"keyboard_name": "Little Big Scroll 4",
+	"manufacturer": "Tweetys Wild Thinking",
+	"url": "https://lectronz.com/stores/tweetys-wild-thinking",
+	"maintainer": "TweetyDaBird",	
+	
+	"processor": "atmega32u4",
+	"bootloader": "caterina",
+	
+	"features": {
+		"bootmagic": true,
+		"command": false,
+		"console": false,
+		"extrakey": true,
+		"mousekey": true,
+		"nkro": false,
+		"rgblight": true,
+		"encoder": true
+	},
+
+	"build": {
+		"lto": true
+	},
+	
+	"debounce": 5,
+	
+	"usb": {
+		"vid": "0xFEED",
+        "pid": "0x23B0",
+		"device_version": "1.1.0"
+	},
+
+	"matrix_pins": {
+        "direct": [
+			[null, "F4"],
+			["E6", "D7"],
+			["B5", "B4"]
+        ]
+    },
+
+	"encoder": {
+        "rotary": [
+            { 
+				"pin_a": "B1", 
+				"pin_b": "F7",
+				"resolution": 2
+			}
+        ]
+    },
+	
+	"rgblight": {
+        "led_count": 6,
+        "pin": "D2",
+        "hue_steps": 8,
+        "saturation_steps": 17,
+        "brightness_steps": 17,
+		"max_brightness": 175,
+        "animations": {
+            "all": true
```

Could you explicitly define which animations are enabled? 
```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```